code/ai/aicode.cpp

```diff
@@ -6159,6 +6168,57 @@ int check_ok_to_fire(int objnum, int target_objnum, weapon_info *wip)
 	return 1;
 }
 
+//	--------------------------------------------------------------------------
+//  Returns true if *aip has a line of sight to its current target.
+//	This is a computationally expensive operation, so use sparingly
+int check_los(int objnum, int target_objnum) {
```

Comment:
Since we are now using C++ conventions, please change the return type of this function to `bool`.

Reply:
To add on, you can mostly ignore the style and conventions of the existing code.  It's not worth any of our coder's time to switch every NULL to nullptr, for instance, but every time we add code, we use nullptr.  Any thing that you can think of to make your code more modern, do it.